packages/mobile/src/components/usdc-manual-transfer-drawer/USDCManualTransferDrawer.tsx

```diff
@@ -35,6 +39,19 @@ const messages = {
   copied: 'Copied to Clipboard!'
 }
 
+const useCreateUserbankIfNeeded = () => {
```

Comment:
I think this could be in common.

Also have a weird feeling about this hook... I wonder if we even need a fee payer override? Is it possible the solanaWeb3Manager doesn't have one? I feel like a useEffect with empty deps would be better to prevent RPC spam

_Also_ also, should this hook be in common?